### PR TITLE
Update misc-vee-validate.vue

### PR DESCRIPTION
**Pull request by iqre8**
Add a regex example where the rules need to change into an rulesObject

## Description

Add an example of regex and change the rule to object

It makes it easier for a new developer to see how to implement regex

## Motivation and Context

It provides an example on how to use regex

## How Has This Been Tested?

I was unable to implement a unit test

I have not tested the changes

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Review by iqre8 — COMMENTED** [review]
Added a dangling comma

File: packages/docs/src/examples/v-form/misc-vee-validate.vue [hunk 1]
```diff
@@ -17,6 +17,23 @@
...
         ></v-text-field>
       </validation-provider>
+      <validation-provider
+        v-slot="{ errors }"
+        name="phoneNumber"
+        :rules="{
+          required: true,
+          digits: 7,
+          regex: '^71|72|74|76|81|82|84|85|86|87|88|89\d{5}$'
+          }"
```
Comment (MajesticPotatoe) [inline packages/docs/src/examples/v-form/misc-vee-validate.vue:27]:
```suggestion
        }"
```

**Review by MajesticPotatoe — DISMISSED** [review]
Doesn't seem to function properly. No error message is displayed when there is more or less 7 digits (only when its empty). Regex match doesn't seem to be firing either (throws error: No such validator 'digits' exists.), also needs to be lint fixed.

**Review by iqre8 — COMMENTED** [review]
Fixed the regex
Escaped \

**Review by iqre8 — COMMENTED** [review]
Fixed the lint issues

**Review by iqre8 — COMMENTED** [review]
Removed extra spaces

**Comment by iqre8** [comment]
@MajesticPotatoe please check if the regex now works properly.

**Comment by MajesticPotatoe** [comment]
`No such validator 'digits' exists` error is still there, likely due to `digits: 7,` and no rule being defined

**Review by iqre8 — COMMENTED** [review]
Added digits

**Review by MajesticPotatoe — APPROVED** [review]
👍 
Thank you for your contribution and interest in improving Vuetify! Make sure to join us in the [Discord community](https://community.vuetifyjs.com).

**Pull request merged**